### ISSUE TITLE
NodeSchool International Day 21st May

### ISSUE THREAD

**Issue by finnp** [nodeschool/berlin#38]
- [x] **Pick location** (Mozilla!)
- [x] Food sponsoring
- [x] Decide on number of participants (40-50 signups)
- [x] Create sign up page earlier (maybe already May 1st?)
- [x] Order stickers
- [x] :school: :bear: Post event on nodeschool.io
- [x] :blue_book: Create facebook event
- [x] :speech_balloon:  Post to slack channel 
- [x] :octocat: Inform on GiHub
- [x] :monkey_face: :envelope: Inform about event through mail chimp
- [x] :bird: Tweet about the event
- [x] :house: Update homepage
- [x] :alarm_clock: Send reminder to participants at the day of event
- [x] :love_letter: Write a follow up mail

**Saturday 21st May 2016**
Additionally see:
https://github.com/nodeschool/international-day/blob/2016/PARTICIPATING.md
https://github.com/nodeschool/international-day

TinkerSoup

**Comment by Realtin** [comment]
I'd really like to do a series of little lightning talks at the beginning and in the middle of the day. At ClojureBridge that was really cool and a nice change in between. 
Something like:
- 10 cool Thing you can do With Node
- Intro to Express or whatever
or little inspirational talks about what ever you like.

**Comment by finnp** [comment]
So we're probably going to do some kind of Tessel hacking as well. Mozilla might provide additional electronics that we can use for this, which is pretty cool.

**Comment by johannhof** [comment]
Yup, we need to figure out how to budget this, but it might be possible. Looping in @mikehenrty

**Comment by mikehenrty** [comment]
Let's figure out exactly what we would want. Ie. breadboards, jumper wires, LEDS, voltemeters, etc. Then we can look into prices (maybe cheaper if we can find bundles or kits). Then we simple ask George Roter. I can handle that last part.

**Comment by finnp** [comment]
Okay, so let's try to make a list. I already looked for bundles/kits, but didn't find anything (there used to be a good one made by fritzing, but they don't sell them anymore).

The basics are what I think we should get in any case (open for debate surely). From the inputs we could choose some, but they should also be fairly cheap. And the cool stuff are nice to have things, where also maybe just having 1 of them would be interesting.

#### Basics
- 10 breadboards
- ~50 LEDs in red, green and blue
- 50 jump wires
- bunch of resistors (depending on what the other parts need)

#### Inputs
- LDR
- Temperature Sensor
- Push-Buttons
- Potentiometer

#### Cool stuff
- Multimeter
- 8x8 LED-Matrix display
- DC motor + motor driver
- USB webcam (used one?)
- basically all official [tessel components](https://tessel.io/modules#tessel-modules) (fairly expensive)

**Comment by mikehenrty** [comment]
yeah, those tessel modules look pretty expensive. do you know if we can use arduino sensors instead? i'm sure it's possible, the question is: does it make hooking them up and talking to them through the node api too hard for the nodeschool environment?

@finnp i don't see actual tessel boards in your list. do we need to order those as well?

also, we have a bunch of cool stuff at the office (neo pixels, sensors, multimeter, motors, firefox os phones that can act like a web cam, etc). if you want to stop by again at some point and see what you want to use, that would be cool.

it would be also great to have a price the hardware bits will cost for this event. when we have that, the approval process will be fairly straightforward. i assume you are handing buying the items, and then we reimburse you, but let me know if i can help out with that.

**Comment by johannhof** [comment]
@mikehenrty we already have the Tessels and just need the components

**Comment by finnp** [comment]
Yes. The list of stuff that you have at the office sounds pretty neat. They should all be usable with the tessels and it would be interesting to see what's working good and is easy to set up.

Generally arduino sensors should be fine, but we would have to make sure that they are easy to connect and use.

I'll try to come up with a budget once we have a better idea about what we need.

**Comment by finnp** [comment]
Sign up for the International Day: http://tito.io/nodeschool-berlin/2016 \o/

**Comment by yoshuawuyts** [comment]
❤️❤️❤️

**Comment by finnp** [comment]
Follow up mail sent 💌 

Thanks again to Mozilla and everyone helping out with it. Looking forward to doing something like this again 😄
